ci-scripts/e2e-cypress.sh

```diff
@@ -46,6 +50,24 @@ done
 
 set -- "${POSITIONAL[@]}"
 
+if [[ "${SPLIT}" = true ]]; then
+    echo "Swtiching to different load balancer"
+    
+    CYPRESS_JSON="projects/storefrontapp-e2e-cypress/cypress.json"
+    CYPRESS_CI_JSON="projects/storefrontapp-e2e-cypress/cypress.ci.json"
+    CYPRESS_CI_B2B_JSON="projects/storefrontapp-e2e-cypress/cypress.ci.b2b.json"
+
+    # modify BASE URL in spartacus instance
+    sed 's/20.83.184.244/20.83.178.185/' .env-cmdrc > .tmp && mv .tmp .env-cmdrc || true
+    
+    # modify API URL in cypress config files.
+    sed 's/20.83.184.244/20.83.178.185/' ${CYPRESS_JSON} > tmp0.json && mv tmp0.json ${CYPRESS_JSON} || true
+    sed 's/20.83.184.244/20.83.178.185/' ${CYPRESS_CI_JSON}  > tmp1.json && mv tmp1.json ${CYPRESS_CI_JSON} || true
+    sed 's/20.83.184.244/20.83.178.185/' ${CYPRESS_CI_B2B_JSON}  > tmp2.json && mv tmp2.json ${CYPRESS_CI_B2B_JSON} || true
+
+    sleep 5
+fi
+
```

Comment:
Are there some alternatives to `sed` we could use for more robustness over time?  Could some kind of environment variable be used?

Reply:
I am not aware of any. I'm open to suggestions.

Reply:
Perhaps something can be done by simply overriding a config on the fly.  See "Overriding Options" https://docs.cypress.io/guides/references/configuration#Overriding-Options